README.md

```diff
@@ -213,6 +214,28 @@ $('.todo-item').on('click', function() {});
 [↑ back to Table of Contents](#table-of-contents)
 
 
+## Use delegated events
+
+When a selector is provided, the event handler is referred to as delegated. jQuery bubbles the event from the event target up to the element where the handler is attached and runs the handler for any elements along that path matching the selector.
+
+### Why?
+
+* Using delegated events allows for events to be processed even to elements added to the document later
+* Keeps the scope of the event *bubling* shorter and thus performant.
+
+### How?
+
+``` javascript
```

Comment:
The code example should show the difference between binding each element and delegating from a parent element:

```javascript
$list = $('[todo-list']).first();
$items = $list.find('[todo-item]');

/* avoid: event listener on each item */
$items.on('click', function(event) { /* ... */ });

/* recommended: event delegation on list */
$list.on('click', '[todo-item]', function(event) { /* ... */ });
```